src/fsharp/service/service.fs

```diff
@@ -1324,13 +1324,23 @@ type TypeCheckInfo
             let underlyingTy = stripTyEqnsAndMeasureEqns g ty
             isStructTy g underlyingTy
 
+        let IsValRefMutable (vref: ValRef) =
```

Comment:
nit case. probaby should be: isValRefMutable

Reply:
blegh, agreed